modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ScalaCaskServerCodegen.java

```diff
@@ -115,6 +116,8 @@ public ScalaCaskServerCodegen() {
 
         typeMapping.put("integer", "Int");
         typeMapping.put("long", "Long");
+        typeMapping.put("AnyType", AdditionalPropertiesType);
```

Comment:
what about mapping `AnyType` to https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ScalaSttp4ClientCodegen.java#L133C36-L133C50 ?

which is https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ScalaSttp4ClientCodegen.java#L96

Reply:
I've changed this to just the ujson 'Value'. It was originally a poison value (as in never actually used).

In future it may be nice to support multiple json libs like ScalaSttp4Client, but for now I think the MVP for cask is to use the single opinionated ujson library which works well out-of-the-box w/ cask